generator/city_roads_generator.cpp

```diff
@@ -55,15 +74,18 @@ void CalcRoadFeatureIds(string const & dataPath, OsmIdToBoundariesTable & table,
         ++inCityPointsCounter;
     }
 
-    if (inCityPointsCounter > kInCityPointsRatio * ft.GetPointsCount())
+    double constexpr kPointsRatioInCity = 0.8;
```

Comment:
Предлагаю указать, в комментарии, почему выбрана 0.8.

Reply:
```
    // Our approximation of boundary overestimates it, because of different 
    // bounding boxes (in order to increase performance). So we don't want 
    // match some long roads as city roads, because they pass near city, but 
    // not though it.
```
?